elytron/src/test/resources/org/wildfly/extension/elytron/elytron-transformers-4.0.xml

```diff
@@ -0,0 +1,2 @@
+<subsystem xmlns="urn:wildfly:elytron:5.0">
```

Comment:
same remark

Reply:
No this is testing transformation to 4 not from 4 - by the end this will likely end up having version 6 of the namespace.